temporal/api/cloud/identity/v1/message.proto

```diff
@@ -69,3 +69,37 @@ message User {
     // Will not be set if the user has never been modified.
     google.protobuf.Timestamp last_modified_time = 8;
 }
+
+message APIKey {
+    // The id of the API Key
+    string key_id = 1;
+    // The current version of the API key specification
+    // The next update operation will have to include this version
+    string resource_version = 2;
+    // The API key specification
+    APIKeySpec spec = 3;
+    // The current state of the API key
```

Comment:
Since we're not using proper enumerates, may need to define in docs the string literals expected here

Reply:
addressed.